src/main/java/TaskHandler.java

```diff
@@ -0,0 +1,149 @@
+public class TaskHandler {
+
+    protected Task[] tasks;
+
+    public TaskHandler() {
+        this.tasks = new Task[100];
```

Comment:
you may want to use a constant MAX_TASKS instead of putting a random number here